susecap-workshop/9001_introduction/modul_introduction_9001.md

```diff
@@ -0,0 +1,65 @@
+![](static/adfinis_sygroup_logo.png)
+
+Be smart. Think open source.
+
+# Introduction to SUSE CAP 
+
+
+## What is SUSE CAP
+
+* SUSE **C**loud **A**pplication **P**lattform
+* CloudFoundry on SUSE CaasP or native K8s
+* installed and configured via Helm Charts
+* Hosting of cf applications
```

Comment:
The PaaS word should probably be on the first slide.

I'm commenting on this line since it's also unclear what a "cf application" ist. IMO such a thing doesn't exist... How about "* PaaS for applications" or something similar.

Reply:
instead of Hosting of cf apps

-> PaaS for applications? 
I would love to integrate also "Code to Production deployment" or something similar. Any suggestions?